.circleci/config.yml

```diff
@@ -183,6 +151,7 @@ jobs:
           root: .
           paths:
             - .
+      - notify-slack-develop-fail
```

Comment:
These `notify-slack-develop-fail` changes send slack alerts if the nightly develop job fails